src/main/java/io/dockstore/provision/DOSPluginUtil.java

```diff
@@ -36,8 +40,20 @@
      * @return the scheme, host, and path of the targetPath, or <code>Optional.empty()</code>
      */
     Optional<ImmutableTriple<String, String, String>> splitUri(String dosURI) {
+        System.out.println("splitUri: Parsing URI " + dosURI);
         if (Pattern.compile(":\\/\\/(.+)/").matcher(dosURI).find()){
             List<String> split  = Lists.newArrayList(dosURI.split(":\\/\\/|/"));
+
+            // Find out if the path is of the DOS GUID format
+            // dos://dg.4503/6f9ad7df-3751-4056-9340-aa9448525b54
```

Comment:
Since you're listing the new format here, it might be useful to also list the old format so that the differences can easily be seen.